.github/workflows/ci.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
       - name: Lint golang code
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
```

Comment:
It seems that #792 also works on it. How about letting #792 solve it, and making this pr focus on its own work?

Reply:
ok, i will rebase this pull request after #792 merged.